src/lib/templates.js

```diff
@@ -67,7 +71,9 @@ const templates = {
   `,
   footer: `
     <div class="${footerPrefix}">
-      Search by <a class="${footerPrefix}--logo" href="https://www.algolia.com/docsearch">Algolia</a>
+      <a class="${footerPrefix}--logo" aria-label="Search by algolia" href="https://www.algolia.com/docsearch">
```

Comment:
```suggestion
      <a class="${footerPrefix}--logo" aria-labelledby="Search by Algolia" href="https://www.algolia.com/docsearch">
```
Seems like `aria-labelledby` is preferred over `aria-label` when the value of the label is already visible ([source](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute)).

I wonder if we also need to add the `title` element?